use_cases/README.md

```diff
@@ -35,14 +35,21 @@ In the description of the issue, write one or more high level goals of the use c
 and if possible a list of the questions that could be relevant for
 reaching that goal.
 
```

Comment:
I think we should require question mapping, otherwise I think we will struggle to connect the use cases to metrics.

Reply:
Do you mean removing "if possible" in that sentence?

Reply:
I hope that's what you meant, @sgoggins ;-) I amended the text with a new commit, and I'm merging all of this. We can always amend it later, if needed, but this way we make this visible to everyone.